generate.sh

```diff
@@ -48,13 +49,17 @@ else
 fi
 
 if [ $? -ne 0 ]; then
-  print_danger "Failed to Clone. Repository does not exist.\n"
+  print_danger "Failed to Clone. Either the repository or the specified branch in that repository does not exist.\n"
   exit 4
 fi
 
 cd ${UNIQUEID}
 print_log "Repository Cloned Successfully!\n"
 
+if [ -z "$DOCPATH" ]; then
+  print_danger "DOCPATH isn't specified. Default value of 'docs' is used"
```

Comment:
If a user doesn't uses the advanced configuration checkbox, then It seems this message will be displayed everytime even though if the user has specified a different path in the `.yaydoc.yml` file. In that case this message will be misleading.